### PR TITLE
[New Game]: Black Jack

### PR DESCRIPTION
**Pull request by singh-anushka**
## PR Description 📜

Black Jack Game created.
Fixes [#82](https://github.com/kunjgit/GameZone/issues/82)

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

Black_Jack

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @singh-anushka ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by kunjgit** [comment]
Hey @singh-anushka 👋!
 
  @kunjgit here, 
 We have this annoucement to make 📢!! 
 We have updated our README 🚀 
 Make sure you follow proper guideline to raise your next issues !! 
 Make sure you join our [Discord🕹️](https://discord.gg/fgwk4XZfxG) if you haven't 😇 
  Thank you 💗!

**Comment by singh-anushka** [comment]
@kunjgit Kindly look into the PR as I resolved the merge conflict too.

File: Games/Black_Jack/ReadMe.md
Comment (kunjgit) [inline Games/Black_Jack/ReadMe.md:1]: Hey !
Rename file to README.md
 so it can be detected 👀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Just make sure you add your game entry with game number * in main README file !! 👀 
and rename your own readme !! All other things are fine 🔥

File: Games/Black_Jack/ReadMe.md
Comment (singh-anushka) [reply Games/Black_Jack/ReadMe.md:1]: Done, Please check and merge these PRs too!
[Atari Breakout game](https://github.com/kunjgit/GameZone/pull/302)
[Guess Colour by RGB game](https://github.com/kunjgit/GameZone/pull/312)
[Resolving responsiveness bug of the main website](https://github.com/kunjgit/GameZone/pull/339)
[Chrome Dinosaur game](https://github.com/kunjgit/GameZone/pull/345)

**Comment by singh-anushka** [comment]
Changes done! Please merge this PR @kunjgit
And please check and merge these PRs too!

[Atari Breakout Game](https://github.com/kunjgit/GameZone/pull/302)
[Guess Colour by RGB](https://github.com/kunjgit/GameZone/pull/312)
[Resolve responsiveness bug of the main website](https://github.com/kunjgit/GameZone/pull/339)
[Chrome Dinosaur](https://github.com/kunjgit/GameZone/pull/345)

Also please add the GSSOC23 label along with the difficulty level 3 card to it too while merging. Thank you!

**Review by kunjgit — APPROVED** [review]

**Comment by kunjgit** [comment]
Thank you  @singh-anushka , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**